docs/articles/documentation/test-api/actions/resize-window.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: docs
+title: Resize Window
+permalink: /documentation/test-api/actions/resize-window.html
+---
+# Resize Window
+
+There are two ways to resize the browser window.
+
+* [Setting the Window Size](#setting-the-window-size)
+* [Fitting the Window into a Particular Device](#fitting-the-window-into-a-particular-device)
+
+## Setting the Window Size
+
+```text
+resizeWindow(width, height)
+```
+
+Parameter  | Type    | Description
+---------- | ------- | --------------------------
+`width`    | Numeric | The new width, in pixels.
+`height`   | Numeric | The new height, in pixels.
+
+The following example demonstrates how to use the `resizeWindow` action.
+
+```js
+import { expect } from 'chai';
+
+test('Side menu disappears on small screens', async t => {
+    await t.resizeWindow(200, 100);
+
+    const menu = await t.select('#side-menu');
+
+    expect(menu.style.display).to.equal('none');
+});
+```
+
+## Fitting the Window into a Particular Device
+
+```text
+resizeWindowToFitDevice(deviceName [, portrait] )
+```
+
+Resizes the window so that it fits into the screen of a certain mobile device.
+
+Parameter               | Type    | Description                                                     | Default
+----------------------- | ------- | --------------------------------------------------------------- | -----------
+`deviceName`            | String  | The name of the device as listed [here](http://viewportsizes.com/). |
```

Comment:
AFAIK it's a bad practice to use `here`, `where`, etc. for links. It better be written as `The name of the device as listed on http://viewportsizes.com/`